src/main/java/core/basesyntax/Report.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Report {
+    private int sumBuy;
+    private int sumSupply;
+    private int result;
```

Comment:
this may not be a separate field
you may just have the `getResult()` method that will provide the calculation

Reply:
removed result
added calculation to getResult()